src/translations/sqlb_de.ts

```diff
@@ -2026,18 +2026,18 @@ Alle aktuell in diesem Feld gespeicherten Daten gehen verloren.</translation>
         <location filename="../ExtendedScintilla.cpp" line="63"/>
         <location filename="../ExtendedScintilla.cpp" line="295"/>
         <source>Ctrl+H</source>
-        <translation></translation>
+        <translation>Strg+H</translation>
```

Comment:
```suggestion
        <translation></translation>
```

Keyboard shortcuts get automatically localized by Qt, and translating key names may indeed break them. They show up on the translation file only for the rare occasions that one has to redirect a shortcut to another one in a specific language. Further information: https://github.com/sqlitebrowser/sqlitebrowser/wiki/Translations#dont-translate-key-names-in-shortcuts